interfaces/Referral-App/src/app/services/spreadsheet.service.ts

```diff
@@ -15,7 +15,10 @@ import { environment } from 'src/environments/environment';
 })
 export class SpreadsheetService {
   private spreadsheetURL = environment.google_sheets_api_url;
-  private spreadsheetId = environment.google_sheets_sheet_id;
+  private spreadsheetId = {
+    amsterdam: environment.google_sheets_sheet_id_amsterdam,
+    utrecht: environment.google_sheets_sheet_id_utrecht,
+  };
```

Comment:
Wouldn't it be possible to supply the "spreadsheetId" from 'outside'?
So that this `SpreadsheetService` doesn't need to be aware of the 'fixed' list of ids?

Reply:
I had to choose between supplying the region name and supplying the sheet id. The region named felt cleaner so rolled with that.

Reply:
Yeah, otherwise the 'mapping'-logic would have to live somewhere else as wel..

Hmm.. For now, only these 2 regions.... Lets see if we can get rid of it soon altogether. ;)